src/Collector/ResolvedNodeCollector.php

```diff
@@ -15,7 +15,7 @@
  * @phpstan-import-type CollectedResolvedNodeArray from CollectedResolvedNode
  * @extends AbstractCollector<Node, CollectedResolvedNode, CollectedResolvedNodeArray>
  */
-final class ResolvedNodeCollector extends AbstractCollector
+final class ResolvedNodeCollector extends AbstractCollector implements PHPStanLatteCollectorInterface
```

Comment:
Why do not add this implements directly to AbstractCollector?

Reply:
In this version I just rewrite collectors to collect only resolved nodes and rest collectors are called only on them.

Reply:
Yes but I do no see reason behind this? If all AbstractCollector offsprings implements this interface why do not add it to AbstractCillector? What am I missing?